### PR TITLE
Add completions for fish

### PR DESCRIPTION
**Pull request by ammgws**

### CODE REVIEW

File: completions/fish/grim.fish [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function complete_outputs
+    if pidof sway; and command -sq jq
```
Comment (emersion) [inline completions/fish/grim.fish]: Instead of `pidof sway`, we can check whether `SWAYSOCK` is set.

Comment (emersion) [inline completions/fish/grim.fish]: Style: we use tabs

File: completions/fish/grim.fish [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        return 1
+    end
+end
+
+complete -c grim -s t --exclusive --arguments 'png ppm jpeg' -d 'Output image format'
+complete -c grim -s q --exclusive -d 'Output jpeg quality (default 80)'
+complete -c grim -s g --exclusive -d 'Region to capture: <x>,<y> <w>x<h>'
+complete -c grim -s s --exclusive -d 'Output image scale factor'
+complete -c grim -s c -d 'Include cursors in the screenshot'
+complete -c grim -s h -d 'Show help and exit'
```
Comment (emersion) [inline completions/fish/grim.fish]: Style: missing final newline

**Comment by ammgws** [comment]
I'll check the meson.build later, I might have messed it up.

File: contrib/completions/meson.build [hunk 3]
```diff
@@ -0,0 +1,14 @@
+fish_comp = dependency('fish', required: false)
+
+if get_option('fish-completions')
+	fish_files(
```
Comment (emersion) [inline contrib/completions/meson.build]: I think this should be `fish_files = files(…)` instead.

Comment (ammgws) [reply contrib/completions/meson.build]: Indeed

**Pull request merged**